uwreact_bringup/launch/zeds.launch

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+
+<launch>
+    <arg name="zed_id1"               default="0" />
+
+    <arg name="pose_frame"            default="map" />
+    <arg name="camera_model"          default="0" /> <!-- 0=ZED, 1=ZEDM-->
+    <arg name="serial_number"         default="0" />
+    <arg name="verbose"               default="true" />
+
+    <arg name="resolution"            default="2" /> <!--0=RESOLUTION_HD2K, 1=RESOLUTION_HD1080, 2=RESOLUTION_HD720, 3=RESOLUTION_VGA -->
+    <arg name="frame_rate"            default="30" />
+    <!-- RESOLUTION_HD2K   -> 2208*1242, available framerates: 15 fps.
+         RESOLUTION_HD1080 -> 1920*1080, available framerates: 15, 30 fps.
+         RESOLUTION_HD720  -> 1280*720,  available framerates: 15, 30, 60 fps.
+         RESOLUTION_VGA    -> 672*376,   available framerates: 15, 30, 60, 100 fps. -->
+
+    <arg name="depth_topic"           default="depth/depth_registered" />
+    <arg name="depth_cam_info_topic"  default="depth/camera_info" />
+    <arg name="laserscan_topic"       default="depth/laserscan"/>
+
+    <arg name="base_frame"            default="zed_camera_center"/>
+    <arg name="laserscan_height"      default="10"/>
+    <arg name="range_min"             default="0.45"/>	
+
+    <!-- First ZED camera -->
+    <group ns="zed1">
+        <include file="$(find zed_wrapper)/launch/zed_camera.launch">
+            <arg name="zed_id"                value="$(arg zed_id1)" />
+
+            <arg name="pose_frame"            value="$(arg pose_frame)" />
+            <arg name="camera_model"          value="$(arg camera_model)" />
+            <arg name="serial_number"         default="$(arg serial_number)" />
+            <arg name="resolution"            default="$(arg resolution)" />
+            <arg name="frame_rate"            default="$(arg frame_rate)" />
+            <arg name="verbose"               default="$(arg verbose)" />
```

Comment:
Think these should be `value`, not `default` - Correct me if I'm wrong

Reply:
Just basically copied what the zed-wrapper launch file had so not sure, does make sense as value

Reply:
I would change it to `value`. If you want to be extra safe, give it a run to be sure, but it should probably be `value`

Reply:
**To be safe, not if you want to be safe. Bad advice on my part - Everything should be tested

Reply:
Default still works almost functionally the same, but it really should be value